sklearn/decomposition/fastica_.py

```diff
@@ -563,8 +565,7 @@ def inverse_transform(self, X, copy=True):
         """
         check_is_fitted(self, 'mixing_')
 
-        if copy:
```

Comment:
Oh! So I must've done this pre-emptively I think... I'll check against master once to confirm!